lib/miq_automation_engine/service_models/miq_ae_service_cloud_volume.rb

```diff
@@ -10,5 +10,7 @@ class MiqAeServiceCloudVolume < MiqAeServiceModelBase
     expose :cloud_volume_backups,   :association => true
     expose :cloud_volume_snapshots, :association => true
     expose :attachments,            :association => true
+    expose :attach_volume,          :override_return => nil
+    expose :detach_volume,          :override_return => nil
```

Comment:
Thanks for comment. These model methods raise an error on failure or nothing meaningful on success. Would ```:override_return => true``` work or should we update methods body to return true/false (or something better)?